### PR TITLE
feature(theme diamond): change line height on radios and checkboxes

### PR DESCRIPTION
**Pull request by jimmyfortinx**
Signed-off-by: Jimmy Fortin <jimmy.fortin@corp.ovh.com>

### CODE REVIEW

File: packages/oui-themes/themes.less [hunk 1]
```diff
@@ -7,6 +7,14 @@
   & .oui-header {
     #oui > .theme-diamond-headers();
   }
+
+  & .oui-checkbox .oui-checkbox__label {
+    #oui > .theme-diamond-base()
```
Comment (antleblanc) [inline packages/oui-themes/themes.less]: Missing `;`

File: packages/oui-themes/themes.less [hunk 2]
```diff
@@ -7,6 +7,14 @@
...
+  }
+
+  & .oui-radio__label {
+    #oui > .theme-diamond-base()
```
Comment (antleblanc) [inline packages/oui-themes/themes.less]: Missing `;`

File: packages/oui-themes/themes.less (hunk 1 above)
Comment (jimmyfortinx) [reply packages/oui-themes/themes.less]: Done

File: packages/oui-themes/themes.less (hunk 2 above)
Comment (jimmyfortinx) [reply packages/oui-themes/themes.less]: Done

**Pull request merged**